liota/lib/transports/web_socket.py

```diff
@@ -55,7 +55,7 @@ def connect_soc(self):
         try:
             self.WebSocketConnection(self.url, False)
         except Exception:
-            log.exception("WebSocket exception, please check the WebSocket address and try again.")
+            raise Exception("WebSocket exception, please check the WebSocket address and try again.")
```

Comment:
Traceback was getting printed as the exception was getting logged previously but you wanted it to be removed as per previous review comment. I have included           log.error(traceback.format_exc()) now it will satisfy both the use cases.